tools/topology/topology1/sof-cavs-nocodec.m4

```diff
@@ -52,6 +52,8 @@ ifelse(NCORES, `4',
 define(DMIC_48k_CORE_ID, `0')
 define(DMIC_16k_CORE_ID, `0')
 define(SSP0_CORE_ID, `0')
+define(SSP0_PIPE7_CORE_ID, `1')
+define(SSP0_PIPE11_CORE_ID, `0')
```

Comment:
we could go further and use SSP0_PIPE7_CORE_ID = 1 and SSP0_PIPE11_CORE_ID = 2, i.e. all three pipelines on different cores just for fun.

Reply:
they are already @plbossart the SSP0_CORE_ID is set to 2 by default. so putting pipe 7 on core 1 and pipe 11 on core 0 would mean 3 different cores